### PR TITLE
fix: update consumption instructions

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ A layer for adding extra udev rules to your image. Use this for better hardware
 
 Add this to your containerfile:
 
-    COPY --from=ghcr.io/ublue-os/udev-rules etc/udev/rules.d/* /etc/udev/rules.d
+    COPY --from=udev-rules:latest /ublue-os-udev-rules /
```
Comment (joshua-stone) [inline README.md]: I believe this should be `--from=ghcr.io/ublue-os/udev-rules:latest`

**Review by bigpod98 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (joshua-stone) [reply README.md]: I think we should also specify how to use the RPM:

```
COPY --from=udev-rules:latest /ublue-os-udev-rules.noarch.rpm /
RUN rpm -ivh /ublue-os-udev-rules.noarch.rpm
```

**Review by joshua-stone — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bigpod98) [reply README.md]: nice catch

**Comment by castrojo** [comment]
I accidentally made an entirely new branch, so closing this, sorry for the inconvenience.

**Pull request closed without merging**